src/embed.js

```diff
@@ -68,9 +68,8 @@ function embed(el, spec, opt) {
     }
     mode = MODES[parsed.library];
 
-    parsedVersion = parsed.version.replace(/^v/g,'');
-    if (versionCompare(parsedVersion, VERSION[mode]) !== 0 ){
-      console.warn("The input spec uses \"" + mode + "\" " + parsedVersion + ", "
+    if (versionCompare(parsed.version, VERSION[mode]) <= 0) {
```

Comment:
If it's 0, then the versions match exactly, right? In that case, we want to not throw a warning.